pkgs/applications/kde/kdenlive/default.nix

```diff
@@ -80,15 +81,19 @@ mkDerivation {
     kpurpose
     kdeclarative
     wrapGAppsHook3
+    glaxnimate
   ];
   # Both MLT and FFMpeg paths must be set or Kdenlive will complain that it
   # doesn't find them. See:
   # https://github.com/NixOS/nixpkgs/issues/83885
-  patches = [ ./dependency-paths.patch ];
+  patches = [ ./dependency-paths.patch ./dependency-paths-glaxnimate.patch ];
 
-  inherit mediainfo;
-  ffmpeg = ffmpeg-full;
-  mlt = mlt-full;
+  # Needed to replace @glaxnimate@ by its path
```

Comment:
This comment does not explain what all the other things are for.

Reply:
removed in https://github.com/NixOS/nixpkgs/pull/324552